packages/expo-dev-launcher/plugin/src/withDevLauncherAppDelegate.ts

```diff
@@ -223,7 +223,7 @@ function removeDevMenuInit(appDelegate: string): string {
 function addDeepLinkHandler(appDelegate: string): string {
   if (!appDelegate.includes(DEV_LAUNCHER_APP_DELEGATE_ON_DEEP_LINK)) {
     appDelegate = appDelegate.replace(
-      'return [RCTLinkingManager application:application openURL:url options:options];',
+      'return [super application:application openURL:url options:options] || [RCTLinkingManager application:application openURL:url options:options];',
```

Comment:
Should test against both or something to support more versions.

Reply:
can you clarify what you mean by this? test what against both?